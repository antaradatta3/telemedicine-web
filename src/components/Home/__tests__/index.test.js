import React from 'react';
import { mount } from 'enzyme';

import Home from '../index';

describe('Component: Home', () => {
  const props = {
    label: 'Sample',
    type: 'primary',
    count: 200,
  };
  it('Home matches snapshot', () => {
    const component = mount(<Home {...props} />);
    expect(component).toMatchSnapshot();
  });
});
