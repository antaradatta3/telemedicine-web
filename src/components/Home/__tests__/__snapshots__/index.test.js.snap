// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Home Home matches snapshot 1`] = `
<Home
  count={200}
  label="Sample"
  type="primary"
>
  <div>
    <div>
      <ThreadList>
        <div>
          <div />
        </div>
      </ThreadList>
    </div>
    <div>
      <Thread>
        <div />
      </Thread>
    </div>
  </div>
</Home>
`;
